_includes/tickets.html

```diff
@@ -87,14 +86,21 @@ <h3><strong>{{category.type}}</strong></h3>
                 <a href="https://www.townscript.com/e/pycon-india-2019/booking" target="_blank" rel="noreferrer noopener" class="dwd-link">
                   <strong>BUY TICKETS</strong>
                 </a>
-	      {% else %}
-		<a href="https://goo.gl/forms/GdtTebWL7mn3oKtW2" target="_blank" rel="noreferrer noopener" class="dwd-link">NOTIFY ME!</a>
+              {% else %}
+                <a href="https://goo.gl/forms/GdtTebWL7mn3oKtW2" target="_blank" rel="noreferrer noopener" class="dwd-link">NOTIFY ME!</a>
               {% endif %}
             </p>
           </div>
         </div>
       {% endif %}
     {% endfor %}
+
+    <div class="row wow fadeIn">
```

Comment:
Not really sure about how to do this in Jekyll itself. Keeping this out of the normal logic for now in mobile view alone.